Genkan cannot load in config.json correctly.

Issue occurs when attempting to run Genkan. Works fine on original development environment but doesn't work when recloned. Possibly a path error.

Files in question:
- `config.json`
- `genkan/config.js`

Possible fix: Don't use relative file path.

```
Error: Cannot find module './config.json'
Require stack:
- /Users/dylankok/Documents/Github Repositories/Genkan/genkan/config.js
- /Users/dylankok/Documents/Github Repositories/Genkan/app.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/dylankok/Documents/Github Repositories/Genkan/genkan/config.js:72:20)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:903:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/dylankok/Documents/Github Repositories/Genkan/genkan/config.js',
    '/Users/dylankok/Documents/Github Repositories/Genkan/app.js'
  ]
}
```

--- a/genkan/config.js
+++ b/genkan/config.js
@@ -7,6 +7,9 @@ require("./auth/tokenGenerator")
 // JSON writer
 const jsonfile = require('jsonfile')
 
+// For getting absolute path of configuration file
+const appRoot = require("app-root-path")
+
 // Logging
 const log = require('loglevel')
 const prefix = require('loglevel-plugin-prefix')
@@ -60,17 +63,15 @@ const generateNewConfig = () => {
 }
 
 try {
-    const config = require("./config.json")
+    const config = require(appRoot + "/config.json")
     module.exports = config
 } catch (error) {
     log.error("Genkan configuration file missing. Generating a new one for you.")
     generateNewConfig()
 
-    const config = require("./config.json")
+    const config = require(appRoot + "/config.json")
     module.exports = config
 }
-
-generateNewConfig()
 
 // if (fs.existsSync("../config.json") === undefined) {
 //     if (fs.existsSync("genkan.lock") === undefined) {

--- a/genkan/config.js
+++ b/genkan/config.js
@@ -54,7 +54,7 @@ const generateNewConfig = () => {
             "password": "password",
             "mailFromAddress": "accounts@example.com"
         },
-        "loggingLevel": "info"
+        "loggingLevel": "debug"
     }
 
     const file = './config.json'

--- a/genkan/config.js
+++ b/genkan/config.js
@@ -1,6 +1,3 @@
-// Filesystem
-const fs = require("fs")
-
 // Press any key to continue
 const pressAnyKey = require('press-any-key');
 

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/HakkouHQ/Genkan#readme",
   "dependencies": {
+    "app-root-path": "^3.0.0",
     "bcrypt": "^5.0.1",
     "chalk": "^4.1.0",
     "compression": "^1.7.4",

